README.md

```diff
@@ -74,19 +97,57 @@ you need to add a CSS class? You would have edit elements through
 both your HTML templates and Javascript controllers. This is where 
 controllers start to become confusing and unlogical.
 
-## The ViewModel in MVVM
+## The MVVM Pattern
 
-The *ViewModel* aims to solve problems that MVC introduces by moving
-binding away from the Javascript and back into HTML. A rewrite of
-the above example in Angular would look like:
+The Model-View-ViewModel or MVVM patterm aims to solve problems 
+that MVC introduces by moving template logic away from the Javascript 
+and back into HTML. A rewrite of the above example in Angular would 
+look like:
 
-```
+```html
 <form ng-submit="doSubmit()"></form>
-<div id="#people">
-  <div class="person" ng-repeat="person in people"></div>
-</div>
+<table>
+  <thead>
+    <th>Name</th>
+    <th>Email</th>
+  </thead>
+  <tbody id="people">
+    <tr ng-repeat="person in people">
+      <td>{{ person.name }}</td>
+      <td>{{ person.email }}</td>
+    </tr>
+  </tbody>
+</table>
 ```
 
-You can see that there is no Javascript involved and our HTML logic
-goes right back to the view. The code is much cleaner and a lot
-easier to write.
+You don't necessarily need to know what this code means just yet, but you can 
+see that there is no Javascript involved and our HTML goes right back to 
+the view. The code becomes much cleaner and a lot easier to write.
+
+### The ViewModel
+
+The ViewModel controls and maintains states of your view and data. It serves
+as a mediator for communication between your view and your controllers. This
+can be a handful at first, but lets imagine it as a real world secretary to
+better understand it.
+
+1. The big boss tells the secretary to inform him of any calls received
+2. The secretary picks up a call and records the purpose of the call
+3. The secretary then tells the big boss about the call
+
+In our example, the:
+
+* secretary is the _ViewModel_
+* big boss is the _controller_
+* the phone is the _view_
+
+### Binding
+
+Binding is a way of telling the ViewModel to run code based on certain
```

Comment:
Also I would include a concrete example here as well. So something like "look at code x, it will change the color of blah based on blah event from the user. We can say that the color is bound to the textfiled" Or something like that. This is all very abstract which makes it a bit hard to follow